lib/vandelay/integrations/vendor/one.rb

```diff
@@ -0,0 +1,27 @@
+module Vandelay
+    module Integrations
+      module Vendor
+        class One < Vandelay::Integrations::Vendor::Base
+            def authenticate
+                super("/auth/1")
+            end
+
+            def patient(id)
```

Comment:
I am returning the patient data from patient method. Now when I am thinking, this is not required as the data is already saved in the parent instance variable `@patient_data`